playlist.c

```diff
@@ -1386,9 +1389,10 @@ void playlist_write_runtime_file(playlist_t *playlist)
    JSON_Writer_WriteNewLine(context.writer);
    JSON_Writer_Free(context.writer);
 
-   playlist->modified   = false;
-   playlist->old_format = false;
-   playlist->compressed = false;
+   playlist->modified        = false;
+   playlist->old_format      = false;
+   playlist->compressed      = false;
+   playlist->cached_external = false;
```

Comment:
The `playlist->cached_external = false;` should be removed here

Reply:
Totally, thanks for that!